pkg/diff/sql_generator.go

```diff
@@ -1461,6 +1514,75 @@ func (s sequenceOwnershipSQLVertexGenerator) GetDeleteDependencies(_ schema.Sequ
 	return nil
 }
 
+type extensionSQLVertexGenerator struct{}
```

Comment:
Since we're not generating SQLVertexes, you can rename this to `extensionSQLGenerator` and get rid of the dependency related methods